src/Core/Content/Product/SearchKeyword/ProductSearchTermInterpreter.php

```diff
@@ -243,6 +243,10 @@ private function score(array $tokens, array $matches): array
             }
 
             foreach ($tokens as $token) {
+                if (false !== stripos($match, $token)) {
```

Comment:
Can you add a test case that verifies the scoring implementation to the existing `\Shopware\Tests\Integration\Core\Content\Product\SearchKeyword\ProductSearchTermInterpreterTest`?

That way it probably also would be easier to understand what effect your changes will have